spec/knapsack_pro/runners/queue/cucumber_runner_spec.rb

```diff
@@ -117,36 +117,50 @@
 
         expect(ENV).to receive(:[]=).with('KNAPSACK_PRO_BEFORE_QUEUE_HOOK_CALLED', 'true')
 
-        expect($?).to receive(:exitstatus).and_return(exitstatus)
+        expect($?).to receive(:exited?).and_return(process_exited)
+        allow($?).to receive(:exitstatus).and_return(exitstatus)
       end
 
-      context 'when tests are passing' do
-        let(:exitstatus) { 0 }
-
-        it 'returns exit code 0' do
-          expect(subject).to eq({
-            status: :next,
-            runner: runner,
-            can_initialize_queue: false,
-            args: args,
-            exitstatus: exitstatus,
-            all_test_file_paths: test_file_paths,
-          })
+      context 'when system process finished its work (exited)' do
+        let(:process_exited) { true }
+
+        context 'when tests are passing' do
+          let(:process_exited) { true }
```

Comment:
no need to duplicated here `let(:process_exited) { true }`. It's already defined in higher context.

Reply:
yeah, forgot to remove, thanks.